.echidna.test.js

```diff
@@ -0,0 +1,26 @@
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+const contract = process.argv[2];
+
+const exec = commands => {
+  try {
+    execSync(commands, { stdio: 'inherit', shell: true }); // Specify the shell if needed
+  } catch (error) {}
+};
+
+const echidnaPath = '/usr/local/bin/echidna';
```

Comment:
This does not work for me. I think the command can be formed using `echidna` command instead of the path, and let the user set it accessible via `$PATH` env var